avaframe/ana3AIMEC/dfa2Aimec.py

```diff
@@ -0,0 +1,131 @@
+"""
+    This is a helper function to export required data from com1DFA to be used by Aimec.
+
+    This file is part of Avaframe.
+"""
+
+# Load modules
+import os
+import glob
+import logging
+import numpy as np
+import shutil
+
+# create local logger
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def makeAimecDirs(avaDir):
+    """ Make directories where Aimec reads input data from """
+
+    # Set directories for Aimec inputs
+    workDir = os.path.join(avaDir, 'Work/ana3AIMEC/com1DFA')
```

Comment:
We are going to run into cross plattform problems if we keep using the / in paths. 
So this code should be
os.path.join(avaDir, 'Work', 'ana3AIMEC', 'com1DFA')

Reply:
Adjusted